### ISSUE TITLE
New task: Counting repetitions

### ISSUE THREAD

**Issue by cornagli** [ixaxaar/pytorch-dnc#49]
Hi,
I am trying to figure out which is the DNC's potentiality. Apparently it looks quite powerful.

I am trying to implement a new task by myself relying on the [pytorch-dnc](https://github.com/ixaxaar/pytorch-dnc) implementation. I am interested in understanding if a DNC can solve it. I would really appreciate any feedback from you, thanks.

### Task description
The task is to count the total number of repeated numbers in a list.

For example:
```
Input: [ 1, 2, 3] 
Output: [0]

Input: [ 1, 2, 3, 2, 4, 1, 5]
                  X     X     : Repetitions
Output: [2]
```
If a DNC is capable of solving this task, I am wondering if it can succeed in generalizing the example counting the repetitions of numbers never encountered during the training process.

### Settings
Assuming that the DNC can solve the task (I suppose a simple LSTM net can), I would structure the data as follows:

- Input: (1, random length, 1) tensor
- Output: Either  (1, random length, 1) tensor or scalar containing the sum of the repetitions
- Loss: depending on the output structure, a square loss function element by element or between two scalars
- DNC parameters: Currently it is obscure to me how to set the memory parameters (word size, number of words) 

 What do you think? Do you think that it would be feasible for the DNC to solve this specific task?

Thanks,
Alessandro
